Dockerfile

```diff
@@ -1,75 +1,39 @@
-		# Replace / rewrite some URLs so that links in the archive go to the correct
-	  # location. Note that the order in which these replacements are done is
-	  # important. Changing the order may result in replacements being done
-		# multiple times.
-		# First, remove the domain from URLs that include the domain
-		&& BASEURL="$VER/" \
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#href="http://docs-stage.docker.com/#href="/#g' \
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#src="https://docs-stage.docker.com/#src="/#g' \
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#href="https://docs.docker.com/#href="/#g' \
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#src="https://docs.docker.com/#src="/#g' \
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#href="http://docs.docker.com/#href="/#g' \
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#src="http://docs.docker.com/#src="/#g' \
-		\
-		# Substitute https:// for schema-less resources (src="//analytics.google.com")
-		# We're replacing them to prevent them being seen as absolute paths below
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#href="//#href="https://#g' \
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#src="//#src="https://#g' \
-		\
-		# And some archive versions already have URLs starting with '/version/'
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#href="/'"$BASEURL"'#href="/#g' \
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#src="/'"$BASEURL"'#src="/#g' \
-		\
-		# Archived versions 1.7 and under use some absolute links, and v1.10 uses
-		# "relative" links to sources (href="./css/"). Remove those to make them
-		# work :)
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#href="\./#href="/#g' \
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#src="\./#src="/#g' \
-		\
-		# Create permalinks for archived versions
-		\
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#href="/#href="/'"$BASEURL"'#g' \
-		&& find target/${VER} -type f -name '*.html' -print0 | xargs -0 sed -i 's#src="/#src="/'"$BASEURL"'#g'; \
-  done; \
-  rm -rf archive_source
-
-# This index file gets overwritten, but it serves a sort-of useful purpose in
-# making the docs/docs-base image browsable:
-
-COPY index.html target
-
-# Serve the site (target), which is now all static HTML
-
-CMD echo "Docker docs are viewable at:" && echo "http://0.0.0.0:4000" && nginx -g 'pid /tmp/nginx.pid;'
+# Start over with nginx:alpine
+FROM nginx:alpine AS docs_base
+
+# Set TARGET to the default location where Nginx serves files from
+# (and where they are stored in all the archives)
+ENV TARGET=/usr/share/nginx/html
```

Comment:
nit: we could change this to an `ARG`, then the `TARGET` env-file isn't persisted in the image